Tasks/AzureFunctionAppV1/task.json

```diff
@@ -17,7 +17,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 200,
+        "Minor": 201,
```

Comment:
This is not correct. It should be what https://whatsprintis.it/ tells you.
In this case it should be 203.

Reply:
Thank you. I have fixed this.